.azure-pipelines/client.yml

```diff
@@ -44,6 +44,15 @@ jobs:
       vmImage: 'vs2017-win2016'
 
     steps:
+      - script: |
+          pip install setuptools wheel 
+          pip install doc-warden
+        displayName: 'Install Warden'
+
+      - script: |
```

Comment:
Any reason you are doing these in multiple steps here where you are doing in one script step in other repos?

Reply:
No particular reason. Just have it cleaned up like this across the other 3 repos.